File: remidial - 2/my sol/Poll/Question.java
import java.util.*;
class Question{
	String question;
	String[]options;
	Hashtable<String,Integer> ht = new Hashtable<String,Integer>();
	
	public Question(String question,String[]options){
		this.question = question;
		this.options = options;

	}
	 public String getQuestion(){
	  return question;
	 }
	 
	 public void setQuestion(String question){
	 	this.question = question;
	 
	 }
	  public String[] getOptions(){
	   return options;
	  }
	  
	  public void setOptions(String[] options){
	   this.options = options;
	  }

	  public void setOptionVotes(int option){
	  	int count=0;
  		// System.out.println(this.options[option]);
  		// System.out.println((ht.containsKey(this.options[option])));
		if (ht.containsKey(this.options[option])) {
			count = ht.get((this.options[option]));
			ht.put(this.options[option], count+1);
		} else {
			ht.put(this.options[option], 1);
		}
		// System.out.println(ht);

	  }
	  public int indexOf(String option){
	  	for (int i = 0;i < options.length;i++) {
	  		if (option.equals(this.options[i])) {
	  			return i;
	  		}
	  	}
	  	return -1;
	  	
	  }
	  public String getText(){
	  	return this.question;

	  }
	  public String commonSelectedOption(){
		String maxKey=null;
		Integer maxValue = Integer.MIN_VALUE; 
		for(Map.Entry<String,Integer> entry : ht.entrySet()) {
		     if(entry.getValue() > maxValue) {
		         maxValue = entry.getValue();
		         maxKey = entry.getKey();
		         // System.out.println(maxValue);
		     }
		     
		}
		return maxKey;

	  }

}

File: remidial - 2/my sol/Poll/Question.py
class Question(object):
    """docstring for Question"""
    global result
    result = {} 
    def __init__ (self, question = None,options = []):
        super(Question, self).__init__()
        self.question = question
        self.options = options
        

    def  getQuestion(self):
        return self.question
    def setQuestion(self,newquestion):
        self.question = newquestion
    def getOptions(self):
        return self.options
    def setOptions(self,newoptions):
        self.options = newoptions
        # print("setOptions",self.options)
    def setOptionVotes(self,option):
        count = 0
        if self.options[option] in result:
            count = result.get(self.options[option])
            result[self.options[option]] = count +1
        else:
            result[self.options[option]] = 1
            # print(result)
            
    def indexOf(self,name):
        for i in range(len(self.options)):
            if name == self.options[i]:
                return i
        return -1
    def getText(self):
        return self.question
    def commonSelectedOption(self):
        lv = list(result.values())
        lk = list(result.keys())
        givenvotes = max(lv)
        for name,votes in result.items():
            if votes == givenvotes:
                return name



        